evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -120,6 +124,24 @@ fn sign(private_key: U256, digest: H256, chain_id: U256) -> Result {
     Ok((sig.v, r_bytes, s_bytes).encode().into())
 }
 
+fn create_wallet(private_key: U256, label: Option<String>, state: &mut Cheatcodes) -> Result {
```

Comment:
let's add a bit of docs on what it does, even if it's a bit obvious :D

Reply:
Sounds good to me! Were you thinking about just adding some comments above this? Or just adding some `create_wallet` documentation in the foundry book?